landoapi/patches.py

```diff
@@ -0,0 +1,154 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import annotations
```

Comment:
This file looks like something other than just ruff related refactoring. Was this intentional?

Reply:
Not intentional, already removed. The only functional change in this PR should be in `tests/test_py3_codequality.py`.